src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import Header from './Header';
+import { BrowserRouter } from 'react-router-dom';
+import { beforeEach, describe, expect, it } from 'vitest';
+
+const MockHeader = () => {
+  return (
+    <BrowserRouter>
+      <Header />
+    </BrowserRouter>
+  );
+};
+
+describe('Header component', () => {
+  describe('header component rendering', () => {
+    it('renders logo as a link with correct text', () => {
+      render(<MockHeader />);
+      const logoElement = screen.getByText(/Shows/i);
+      expect(logoElement).toBeInTheDocument();
+
+      const logoLink = screen.getByRole('link');
+      expect(logoLink).toBeInTheDocument();
+    });
+
+    it('renders search form with search input and submit button', () => {
+      render(<MockHeader />);
+      const FormElement = screen.getByTestId('search-form');
+      expect(FormElement).toBeInTheDocument();
+      expect(FormElement).toContainHTML('button');
+      expect(FormElement).toContainHTML('input');
+    });
+  });
+
+  describe('Search component', () => {
```

Comment:
Do you mean there shouldn't be tests for local Storage? I include it there because it is a part of the search component functionality. It should be removed?